service/status-report/main.go

```diff
@@ -215,6 +222,25 @@ func setClusterInformation(ctx context.Context, cl client.Client, status interfa
 
 }
 
+func setStorageQuotaUtilizationRatio(ctx context.Context, cl client.Client, status interfaces.StorageClientStatus) {
+	clusterResourceQuota := &quotav1.ClusterResourceQuota{}
+	clusterResourceQuota.Name = status.GetClientName()
+
+	if err := cl.Get(ctx, client.ObjectKeyFromObject(clusterResourceQuota), clusterResourceQuota); err != nil {
+		// No need to check for NotFound because unlimited quota client will not have CRQ resource
+		if !errors.IsNotFound(err) {
+			klog.Warningf("Failed to get clusterResourceQuota %q: %v", clusterResourceQuota.Name, err)
+		}
+	}
+	if clusterResourceQuota.Status.Total.Hard != nil {
+		var total = clusterResourceQuota.Status.Total.Hard["requests.storage"]
+		var used = clusterResourceQuota.Status.Total.Used["requests.storage"]
+		var ratio = used.AsApproximateFloat64() / total.AsApproximateFloat64()
```

Comment:
what was the issue w/ using the walrus op `:=`?

Reply:
no issue, while typing i missed i guess.

Reply:
no issue by typing end up writing this. will change.